comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/ItemVariantResource.java

```diff
@@ -91,6 +96,15 @@ public WsItemVariant getItemVariant(@PathParam("id") long id) {
         return wsItem;
     }
 
+    @Path("{id}")
+    @DELETE
+    public void deleteItemVariant(@PathParam("id") long id) {
```

Comment:
note to myself: guess we'll need both sometime: delete just after an error, inactive otherwhise.